CONTRIBUTING.md

```diff
@@ -1,5 +1,21 @@
 # Contributing to Node.js
 
+Contributions to Node.js include code, documentation, answering user questions,
+running the project's infrastructure, and advocating for all types of Node.js
+users.
+
+The Node.js project welcomes all contributions from anyone willing to work in
+good faith with other contributors and the community. No contribution is too
+small and all contributions are valued.
```

Comment:
This part is from https://github.com/nodejs/node/commit/7e1b178fb637abc68b1d4da1363a19db7ad02d6c#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055L3